mobile/App.js

```diff
@@ -1,20 +1,34 @@
 import React from 'react';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { createStackNavigator } from '@react-navigation/stack';
 import { NavigationContainer } from '@react-navigation/native';
 import MapScreen from './Screen/MapScreen';
 import GalleryScreen from './Screen/GalleryScreen';
 import SubmissionScreen from './Screen/SubmissionScreen';
+import ProfileScreen from './Screen/ProfileScreen';
+import SignUpScreen from './Screen/SignUpScreen';
 
 const Tab = createBottomTabNavigator();
+const Stack = createStackNavigator();
+
+function HomeStackScreen() {
+  return (
+    <Tab.Navigator>
+      <Tab.Screen name="Map" component={MapScreen} />
+      <Tab.Screen name="Gallery" component={GalleryScreen} />
+      <Tab.Screen name="Submission" component={SubmissionScreen} />
+      <Tab.Screen name="Profile" component={ProfileScreen} />
+    </Tab.Navigator>
+  );
+}
```

Comment:
Great job on using the Stack and Tab navigators! The navigation works as expected and the setup will be re-used if other screens get added to the stack